### DISCUSSION TITLE
Can't upload apk files to releases anymore

### DISCUSSION THREAD

**Discussion by connyduck** [community/community#40098] [Repositories]
### Select Topic Area

Product Feedback

### Body

Since today, I can't upload apk files to releases anymore. Error message: "We don't support this file type"
apk files (Android app packages) are very important though, they are basically the thing being released. Some users prefer to directly download them to using App Stores.
Please enable apk uploads again.

**Comment by ossanna16** [comment]
Hi there @connyduck and welcome to our community! Thank you for reporting this!

**Comment by thomas3456734567** [comment]
If you're facing issues with uploading APK files, it's always helpful to explore reliable cloud storage solutions. I’ve been using [terabox mod apk download](https://teraboxapkdownload.com/) for seamless storage and quick access to my files. It has made managing large files much easier.

**Comment by jhon45678** [comment]
Rate limiting has definitely impacted how often we can run background checks on dynamic content. I faced something similar while automating updates for menu data on brewmenu. You can also [explore more here](https://brewmenuus.com/) if you're interested in real-time menu updates and syncing.

**Comment by jameswrightbrown-prog** [comment]
I faced the exact same issue today while uploading the latest build of one of my Android projects the [moviesmod apk](https://movies-mod.com/) release. GitHub suddenly stopped accepting .apk files and showed the same error:
“We don’t support this file type.”
After trying a few things, here’s what actually worked for me:
1)Rebuilding & Re-signing the APK
When I generated a fresh signed APK, GitHub accepted it without any problem. It seems the validation fails if the file signature is slightly off.
2)Temporary rename workaround
I renamed the file from .apk → .zip and then back to .apk.
Weirdly, after doing this once, the release upload worked normally again.
3)Switching to the old release editor
If you click “Revert to previous version” on the Releases page, the classic uploader allows the APK to upload without throwing the file type error.
So it doesn’t look like GitHub disabled APK uploads more like a temporary validation bug. After rebuilding and renaming, I was able to upload the moviesmod apk update successfully.
Hope this helps anyone running into the same problem.

**Comment by steafeminfi** [comment]
this issue impacts a large number of [Android APK](https://animesalt.org/) developers who rely on GitHub Releases for transparent and trusted distribution. Blocking .apk files without a clear explanation creates friction and forces unnecessary workarounds.

A clear confirmation from the GitHub team—whether this is a bug, a policy change, or a security decision—would go a long way. If permanent, official alternatives or exemptions for verified repositories would be highly appreciated.

Thanks to everyone contributing here. Hoping for an official update soon

**Comment by skyeava974-alt** [comment]
I have been using [Terabox](https://apkteraboxx.com/) for storing and managing Android app files, and it makes sharing them really easy. While it is not a direct replacement for release uploads, it is super convenient for keeping all your builds organized and accessible.

**Comment by dhanifrahman** [comment]
**The Problem**

GitHub hasn't banned APKs; you are likely dropping the file into the wrong box.

**The Fixes**

- Move the file lower: You are probably dragging the APK into the Release Description box (the text area). That box only likes images and PDFs. Instead, scroll down and drop the file into the box labeled "Attach binaries by dropping them here" at the very bottom.

- Check your browser: If you are using the right box and it still fails, try Incognito Mode. Sometimes browser extensions (like ad-blockers) glitch the upload button.

- The "Emergency" fix: If you’re in a hurry, put the APK inside a ZIP file. GitHub almost always accepts ZIPs, regardless of which box you use.

**Comment by fm23mod-blip** [comment]
This is a very helpful and informative platform for users looking for easy access to employee systems. I found the guide on <a href="https://sites.google.com/view/ukglogin/">ukg login</a> really simple and useful. Everything is explained clearly

**Comment by fm23mod-blip** [comment]
This is a really interesting and useful platform  I found the information about <a href="https://sites.google.com/view/bashra/">chatpic</a> quite helpful and easy to understand. The site layout is also clean and user-friendly.
